pytorch_lightning/accelerators/cpu_accelerator.py

```diff
@@ -90,3 +90,7 @@ def sync_tensor(self,
                     group: Optional[Any] = None,
                     reduce_op: Optional[Union[ReduceOp, str]] = None) -> torch.Tensor:
         return tensor
+
+    @property
+    def is_ddp_based(self):
```

Comment:
this seems very specific no?

Reply:
It was and the name wasn't very well defined. I've changed this to `require_distributed_sampler` which all accelerators now have control over, which makes it a lot cleaner when defining custom accelerators!